styles/style.css

```diff
@@ -0,0 +1,680 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    font-family: sans-serif;
+    font-size: 1.3rem;
+}
+
+header {
+    display: flex;
+    flex-direction: row;
+    background-image: url(../images/header-image.jpg);
+    background-repeat: no-repeat;
+    background-size: cover; 
+    width: 100%;
+    height: 375px;
+}
+
+.top-left {
+    display: flex;
+    flex-direction: column;
+    padding: 4rem 0 0 8rem;
+}
+
+div.checkout {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    padding: 4rem 0;
+}
+
+.top-logo {
+    max-width: 100%; 
+    width: 100%;
+}
+
+.slogan {
+    font-style: italic;
+    font-size: 1.7rem;
+    color: #a1c349;
+    margin-top: 0;
+    width: fit-content;
+}
+
+.slogan p {
+    margin: 0;
+}
+
+[src="./images/cart-icon.svg"] {
+    max-width: 100%;
+    width: 5%;
+}
+
+.total {
+    color: #FFF;
+    padding-bottom: 0;
+}
+
+[value="Checkout"] {
+    padding: 0.5rem 2rem;
+    margin-top: 1rem;
+    background-color: #a1c349;
+    color: #FFFfff;
+    font-weight: bold;
+    border-radius: 2px;
+    border: none; 
+}
+
+.main-nav {
+    background-color: rgb(224, 224, 226);
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+}
+
+ul { 
+    font-size: .85em; 
+    color: #ccc; 
+    margin: 1.5rem 0 1.5rem 12rem;
+    padding-left: 0;
+    display: inline-flex;
+}
+
+    li { 
+        list-style-type: none;
+        padding-right: 34px;
+        float: none;
+    }
+
+    li:after {
+        content: "/";
+        color: #0a3d61;
+        font-size: 1.3em;
+        padding-left: 1rem;
+        position: absolute;
+    }
+
+    li:last-of-type:after {
+        content: none;
+    }
+
+    li:last-child {
+        padding-right: 0;
+    }
+
+    a {
+        text-decoration: none;
+        text-transform: capitalize;
+        font-weight: bold;
+        color: #0a3d61;
+    }
+
+    a[href="./blog.html"]{

```

Comment:
Like the svg above yo won't want to target specific files in your css

Reply:
This style has just muted.
Thanks for letting me know.